lib/fluent/supervisor.rb

```diff
@@ -439,14 +439,17 @@ def show_plugin_config
     def supervise
       $log.info "starting fluentd-#{Fluent::VERSION}"
 
+      rubyopt = ENV["RUBYOPT"]
       if Fluent.windows?
         fluentd_spawn_cmd = ServerEngine.ruby_bin_path + " -Eascii-8bit:ascii-8bit "
+        fluentd_spawn_cmd << ' "' + rubyopt.gsub('"', '""') + '" ' if rubyopt
```

Comment:
Shellwords did't work on windows, then I found this 
http://stackoverflow.com/questions/5636601/is-there-an-equivalent-to-the-ruby-shellwords-module-for-the-windows-shell

If I'm misunderstanding, please tell me.